qcut/research/jianying-runtime-probe/render-transition-video.sh

```
@@ -0,0 +1,125 @@
+if ! awk -v value="$TRANSITION_DURATION" 'BEGIN { exit !(value > 0) }'; then
+  printf 'Transition duration must be greater than 0.\n' >&2
+  exit 1
+fi
+if (( width % 2 != 0 )); then
+  width=$((width + 1))
+fi
+if (( height % 2 != 0 )); then
+  height=$((height + 1))
+fi
+
+readonly TEMP_DIR="$(mktemp -d "${TMPDIR:-/tmp}/qcut-jianying-transition.XXXXXX")"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)